### PR TITLE
F#11 screen dumb

### LINKED ISSUE #11: Upload models of screen and dumb
Into cad/ upload .blend file.
Into meshes/ upload visual and collision .dae files.
In models/soft_chair create model.sdf and model.config files.

### PR DESCRIPTION
**Pull request by v-lavrushko**
Models done.
closes #11

### CODE REVIEW

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please remove `robert_gazebo/.DS_Store` and similar files. Add `.DS_Store` to `.gitignore`.

**Comment by AndriyPt** [comment]
Please restore file `robert_gazebo/worlds/ucu.world`

**Comment by sofiia-tesliuk** [comment]
done.

**Comment by sophia-tslk** [comment]
done.

File: robert_gazebo/models/APPS_letters/model.sdf [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <pose>0 0 0 0 0 0</pose>
+        <static>true</static>
+        <link name='model_body'>
+            <visual name="visual">
+                <geometry>
+                    <mesh>
+                        <uri>model://robert_gazebo/meshes/APPS_letters_visual.dae</uri>
+                        <scale>1 1 1</scale>
+                    </mesh>
+                </geometry>
```
Comment (AndriyPt) [inline robert_gazebo/models/APPS_letters/model.sdf]: please add collision and inertia

File: robert_gazebo/models/screen/model.sdf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <link name='model_body'>
+            <visual name="visual">
+                <geometry>
+                    <mesh>
+                        <uri>model://robert_gazebo/meshes/screen_visual.dae</uri>
+                        <scale>1 1 1</scale>
+                    </mesh>
+                </geometry>
+            </visual>
+        </link>
```
Comment (AndriyPt) [inline robert_gazebo/models/screen/model.sdf]: please add collision and inertia

**Review by AndriyPt — CHANGES_REQUESTED** [review]

File: robert_gazebo/models/APPS_letters/model.sdf (hunk 1 above)
Comment (sofiia-tesliuk) [reply robert_gazebo/models/APPS_letters/model.sdf]: This model is static and located on the wall, so robert won't have interaction with it.

Comment (AndriyPt) [reply robert_gazebo/models/APPS_letters/model.sdf]: it is for now. But in future we might have some arm on top of Robert etc. If we are already spending time on models it good to build them with all required attributes :)

**Review by AndriyPt — CHANGES_REQUESTED** [review]

**Comment by AndriyPt** [comment]
@sophia-tslk It still doesn't appear right (letters and TV are half visible). Please move APPS letters and TV to another PR. The rest could remain in the same PR. In the future please add single item per PR
image

**Comment by sofiia-tesliuk** [comment]
This two objects are static and located with z = 0, so another parts of them are under floor.
Letters and puffy were moved to anothers branch.

**Comment by AndriyPt** [comment]
@sophia-tslk please change file access mode from 755 to 644 as it was before using chmod command.
https://www.washington.edu/computing/unix/permissions.html
```bash
chmod 644 <your_file>
```
image

**Comment by sofiia-tesliuk** [comment]
Done

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**